package.json

```diff
@@ -21,8 +21,12 @@
     "atom": ">=0.174.0"
   },
   "dependencies": {
-    "atom-linter": "^3.0.0"
+    "atom-linter": "^3.0.0",
+    "atom-package-deps": "^3.0.0"
```

Comment:
Might as well specify the latest version (although it doesn't really matter right now since all users will be installing a new version, we might as well specify that it works with the latest from the start.)